README.md

```diff
@@ -5,6 +5,36 @@
 
 This repo contains ansible and CI related assets used for the OCP 3 to 4 migration project, the primary use case for these tools is to integrate with Jenkins and allow the creation of all necessary CI workflows. It will consist of several ansible playbooks, which will prepare a environment for migration purposes. This involves provisioning customer-like cluster deployments with [OA installer](https://github.com/openshift/openshift-ansible), and setup of the migration tools from [mig-controller](https://github.com/fusor/mig-controller).
 
+## Migration controller e2e tests
+
+Viz. [parallel-base](https://github.com/fusor/mig-ci/blob/master/pipeline/parallel-base.groovy) pipeline. The execution is triggered by update in mig-controller repo by parametrized job. Parallel base consists of several parameters to choose the type of the deployment (Agnosticd, OA, or origin3-dev).
```

Comment:
Remove the Viz, The execution of the e2e tests is triggered by updates in the mig-controller repo, which will cause Jenkins to use a parameterized trigger job calling the parallel base pipeline. Parameters such as type of deployment (Agnosticd, OA or origin3-dev) amongst others can be set.

Reply:
Sure, good note, will update this in a future PR.